libraries/timeline-view-compose/src/main/java/com/trendyol/uicomponents/timelineviewcompose/ScaleAnimation.kt

```diff
@@ -0,0 +1,40 @@
+package com.trendyol.uicomponents.timelineviewcompose
+
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.graphics.graphicsLayer
+import com.trendyol.uicomponents.timelineviewcompose.model.PointAnimation
+import kotlinx.coroutines.delay
+
+@Stable
+internal fun Modifier.scaleAnimation(pointAnimation: PointAnimation?) = composed {
+    var shouldStartAnimation by remember { mutableStateOf(false) }
```

Comment:
what if we use early return when `pointAnimation` is null?

Reply:
you're right. I fixed it.